### ISSUE TITLE
Does not work anymore

### ISSUE THREAD

**Issue by r2ixuz** [werty1st/Redirect-Click-n-Load#1]
First of all thanks for the great plugin. It served me very well for the last couple of years.

Unfortunately since Chrome 81.0.4044.138 (on macOS 10.15.4) it does not work anymore.

If I setup manually an SSH tunnel I get the links to Pyload correctly transfered, so I think it is your plugin wich has a problem. :-(

**Comment by werty1st** [comment]
Hi,

i can not reproduce the error. do you have more details?
i'm on the same version with chrome and macos.

https://www.youtube.com/watch?v=BLyknZbai5g&feature=youtu.be

**Comment by r2ixuz** [comment]
I don't know. From one day to another it just stoped working. I'm even unsure, if it is really since the last Chrome update.

I already reinstalled the plugin. Tried FQHN, short name, IP and with two Pyload Instances I have running (each in a own Docker container on seperated hosts).

Only thing I'm sure is, that it works with an SSH tunnel or with this extension: (Chromebook Connection Forwarder) https://chrome.google.com/webstore/detail/connection-forwarder/ahaijnonphgkgnkbklchdhclailflinn
